components/ama-rsvp.js

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect } from 'react'
+import {
+  Button,
+  Box,
+  Card,
+  Grid,
+  Heading,
+  Input,
+  Label,
+  Text,
+  Select,
+  Checkbox
+} from 'theme-ui'
+import fetch from 'isomorphic-unfetch'
+
+const AMARsvp = ({ id }) => {
+  const [email, setEmail] = useState('')
+  const [name, setName] = useState('')
+  const [association, setAssociation] = useState('')
+  const [status, setStatus] = useState('')
+  useEffect(() => {
+    setTimeout(() => {
+      setEmail('')
+      setName('')
+      setAssociation('')
+    }, 2000)
+  }, [status])
+  return (
+    <Card sx={{ mt: [3, 4] }}>
+      <Heading variant="headline" sx={{ mt: 0, mb: 1 }}>
+        RSVP for this AMA
+      </Heading>
+      <form
+        action={`/api/rsvp?id=${id}`}
+        onSubmit={e => {
+          e.preventDefault()
+          fetch(`/api/rsvp?id=${id}`, {
+            method: 'POST',
+            data: JSON.stringify({ phone })
```

Comment:
This line will error since `phone` is undefined, & we won’t save any of the data entered on this form